src/sentry/grouping/ingest/config.py

```diff
@@ -79,6 +92,11 @@ def _auto_update_grouping(project: Project) -> None:
             "sentry:secondary_grouping_expiry": expiry,
             "sentry:grouping_config": new_config,
         }
+        # Any project on deprecated configs may have disabled auto updates and
+        # this will reduce the chance of happening unintentionally.
```

Comment:
The chance of _what_ happening unintentionally - being on a deprecated config, or having auto-update False?

Reply:
Being on a deprecated config because they disabled updates. I _assume_ that for testing purposes they followed instructions on how to move to the mobile config and not being moved out of it.

Reply:
Got it. Thanks for clarifying. In that case, how about 

```suggestion
        # Any project on deprecated configs may be there only because auto updates are
        # disabled (not because they want to use that specific config). This will reduce the 
        # chance of that happening unintentionally.
```